shared/bls/herumi/signature_test.go

```diff
@@ -191,3 +190,12 @@ func TestSignatureFromBytes(t *testing.T) {
 		})
 	}
 }
+
+func TestCopy(t *testing.T) {
```

Comment:
This doesn't test that it is a safe copy all it does check that the interface representation is not equal. Also 
the fact that after a copy the different signatures are not equal means that either the copy is bad or the 
interface equality action regards them as different when they are not. For values like these `DeepEqual` is 
more accurate.